sections/featured-product.liquid

```diff
@@ -134,6 +155,45 @@
                     {%- endform -%}
                   </div>
                 {%- endif -%}
+              {%- when 'inventory' -%}
+                {%- if product.selected_or_first_available_variant.inventory_quantity != 0 -%}
+                  <p
+                    class="product__inventory no-js-hidden{% if block.settings.text_style == 'uppercase' %} caption-with-letter-spacing{% elsif block.settings.text_style == 'subtitle' %} subtitle{% endif %}"
+                    id="Inventory-{{ section.id }}"
+                    role="status"
+                    {{ block.shopify_attributes }}
+                  >
+                    {%- if product.selected_or_first_available_variant.inventory_quantity
```

Comment:
Fair enough. I think it's an edge case. We have this logic for other blocks as well. I think we should consider it as a different issue.

Reply:
That depends on what the other blocks do. Setting a threshold for low stock is linked to data (the number of items left available), so it can have different consequences compared to just a visual difference. Indicating "low stock" vs "in stock" in two different places can suggest to buyers that there's an issue somewhere. I think that kind of data should be consistent across a shopping site. @ludoboludo, @lucianosousa, @YoannJailin what do you think?

Reply:
I remember originally Dan and Yoann mentioned that they would like to have a threshold setting in the admin. Until it happens I more lean towards  keeping the feature for both sections but add additional info for the block in Featured product saying something like "Pls make sure to set threshold exactly the same as you set it in Product page"